packages/components/src/vue/components/setting.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div :class="classes" v-if="dataModel && Object.keys(dataModel).length > 0  && !dataModel.hidden">
-    <label class="alpheios-setting__label" v-show="showTitle">{{ labelText}}</label>
+    <label class="alpheios-setting__label" v-show="showLabelText" v-if="labelText">{{ labelText}}</label>
```

Comment:
is there any reason to just not use labelText here, and make it always support HTML?

Reply:
Oh, I didn't think about this, of course - it is much easier :)